src/main/java/data_gateway/CalendarManager.java

```diff
@@ -7,7 +7,7 @@
 
 public interface CalendarManager {
 
-    void addEvent(CalendarEventModel eventData);
+    long addEvent(CalendarEventModel eventData);
```

Comment:
Should the changes to Event, CalendarAnalyzer and Tasks be in another PR?

Reply:
Yea this wasn't my work I don't know how this ended up in my pr and I'm not too sure how to remove it either. Also I think I can move the getCurrentTime function into the ClockController class and I'm looking into the task functionality of javafx right now to handle it

Reply:
might be because you haven't merged main into `pomodoro_gui` (the target branch)